external-plugins/referee/server/server_test.go

```diff
@@ -0,0 +1,294 @@
+	return arguments.Error(0)
+}
+
+type fakeGitHubGraphQLClient struct {
+	mock.Mock
+}
+
+func newFakeGitHubGraphQLClient() *fakeGitHubGraphQLClient {
+	return &fakeGitHubGraphQLClient{}
+}
+
+func (_m *fakeGitHubGraphQLClient) FetchPRTimeLineForLastCommit(org string, repo string, prNumber int) (ghgraphql.PRTimelineForLastCommit, error) {
+	arguments := _m.Called(org, repo, prNumber)
+	return arguments.Get(0).(ghgraphql.PRTimelineForLastCommit), arguments.Error(1)
+}
+
+var _ = Describe("", func() {
+	Context("handlePullRequestComment", func() {
+		var server Server
+		var mockGitHubClient *fakeGitHubClient
+		var mockGitHubGraphQLClient *fakeGitHubGraphQLClient
+		BeforeEach(func() {
+			entry := logrus.StandardLogger().WithFields(map[string]interface{}{"type": "testlogger"})
+			mockGitHubClient = newFakeGitHubClient()
+			mockGitHubGraphQLClient = newFakeGitHubGraphQLClient()
+			server = Server{
+				TokenGenerator:  nil,
+				BotName:         botuser,
+				Log:             entry,
+				GithubClient:    mockGitHubClient,
+				GHGraphQLClient: mockGitHubGraphQLClient,
+				DryRun:          false,
+			}
+		})
+		It("doesn't handle other action", func() {
+			Expect(server.handlePullRequestComment(github.IssueCommentEvent{
+				Action: github.IssueCommentActionDeleted,
+				Issue:  github.Issue{},
+				Comment: github.IssueComment{
+					User: github.User{Login: user},
+				},
+				Repo: github.Repo{},
+				GUID: "",
+			})).ShouldNot(HaveOccurred())
+			mockGitHubGraphQLClient.AssertExpectations(GinkgoT())
+			mockGitHubClient.AssertExpectations(GinkgoT())
+		})
+		It("doesn't handle issue comment", func() {
+			Expect(server.handlePullRequestComment(github.IssueCommentEvent{
+				Action: github.IssueCommentActionCreated,
+				Issue:  github.Issue{},
+				Comment: github.IssueComment{
+					User: github.User{Login: user},
+					Body: `This is an issue comment
+
+/help-wanted
+`,
+				},
+				Repo: github.Repo{},
+				GUID: "",
+			})).ShouldNot(HaveOccurred())
+			mockGitHubGraphQLClient.AssertExpectations(GinkgoT())
+			mockGitHubClient.AssertExpectations(GinkgoT())
+		})
+		It("doesn't handle comment if it doesn't contain a test command", func() {
+			Expect(server.handlePullRequestComment(github.IssueCommentEvent{
+				Action: github.IssueCommentActionCreated,
+				Issue: github.Issue{
+					Number:      prNumber,
+					PullRequest: &struct{}{},
+				},
+				Comment: github.IssueComment{
+					User: github.User{Login: user},
+					Body: `This is a comment on a pull request but not a test command
+
+/help-wanted
+`,
+				},
+				Repo: github.Repo{
+					Owner: github.User{Login: org},
+					Name:  repo,
+				},
+				GUID: "",
+			})).ShouldNot(HaveOccurred())
+			mockGitHubGraphQLClient.AssertExpectations(GinkgoT())
+			mockGitHubClient.AssertExpectations(GinkgoT())
+		})
+		It("fetches number of retests on test all comment, but doesn't place comment since not enough retest comments found", func() {
```

Comment:
nit
```suggestion
		It("fetches number of retests on test-all comment, but doesn't post comment since not enough retest comments found", func() {
```